amundsen_application/static/js/components/common/EditableText/index.tsx

```diff
@@ -162,7 +162,7 @@ class EditableText extends React.Component<EditableTextProps, EditableTextState>
           show={true}
           target={this.getTarget.bind(this,'textArea')}
         >
-          <Tooltip>
+          <Tooltip container={this}>
```

Comment:
I didn't get this quite working in the 5 minutes I played with it, but this is how we can fix the floating tooltips issue.

Reply:
`container` should actually go on the Overlay component.